openshift/README.md

```diff
@@ -73,6 +73,28 @@ The best way to publish an image is to make a new release of the project,
 using the github [dialog](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release).
 This will also generate release notes.
 
+## Update production
+
+Once you created the new release and the image is built and present in [our
+quay repository](https://quay.io/repository/log-detective/website?tab=history),
+you can update the website.
+
+The easiest way is to delete the existing running pod (yes, scary 😱) and let
+OpenShift fetch the newest image and use it to start a new pod.
+
+```
+oc get pods
+oc delete pod log-detective-website-xxxxxxxx-xxxxx
+```
```

Comment:
Would `oc rollout restart deploy/log-detective-website` work instead of deleting?

Reply:
from the `--help` page:
```
 There are several deployment strategies defined:

  *  Rolling (default) - scales up the new replication controller in stages, gradually reducing the number of old pods.
If one of the new deployed pods never becomes "ready", the new rollout will be rolled back (scaled down to zero). Use
when your application can tolerate two versions of code running at the same time (many web applications, scalable
databases)
  *  Recreate - scales the old replication controller down to zero, then scales the new replication controller up to
full. Use when your application cannot tolerate two versions of code running at the same time
```

yes, that should work and we already have that documented on the lin 130

https://github.com/fedora-copr/log-detective-website/pull/111/files#diff-10a0fba9e0f5927439e2c632e20443232a4578c9be2df85bd25f6f768ebf5c1eR130